drf_braces/fields/modified.py

```diff
@@ -24,6 +24,24 @@ def quantize(self, value):
         return super(DecimalField, self).quantize(value)
 
 
+class CurrencyField(DecimalField):
+    """
+    Currency field subclass of Decimal used for rounding currencies
+    to two decimal places.
+    """
+    precision = 2
+
+    def __init__(self, max_digits=None, *args, **kwargs):
+        super(DecimalField, self).__init__(
+            max_digits=max_digits,
+            decimal_places=None,
+            *args, **kwargs
+        )
+
+    def to_internal_value(self, data):
+        return round(data, self.precision)
```

Comment:
`super` implementation does a bunch of other useful checks. lets instead adjust `self.quantize` and `self.validate_precision`